.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -23,8 +23,20 @@ A clear and concise description of what you expected to happen.
 If applicable, add config files to help explain your problem.
 
 **Environment**
-- Cloud provider: [AWS | Azure | Any]
+- Cloud provider: [AWS | Azure | All | None]
 - OS: [e.g. Ubuntu 18.04.4 LTS, you can use `cat /etc/os-release`]
```

Comment:
Or more generally cat /etc/*release should work on Ubuntu/RH/CentOS, but it's not part of your change

Reply:
It seems `cat /etc/os-release` works on all supported distros.